capa/features/extractors/base_extractor.py

```diff
@@ -262,3 +262,88 @@ def extract_insn_features(
           Tuple[Feature, Address]: feature and its location
         """
         raise NotImplementedError()
+
+
+@dataclass
+class ProcessHandle:
+    """
+    reference to a process extracted by the sandbox.
+
+    Attributes:
+        ppid: parent process id
+        pid: process id
+        inner: sandbox-specific data
+    """
+
+    ppid: int
+    pid: int
```

Comment:
hm, now that i think about this: this is the abstract interface for dynamic handles, which means that all implementors *must* be able to provide this data. if we're not sure that its available, then we should rely on implementors to subclass and/or use the `inner` property.

with that in mind, i think we should at least remove ppid. im not sure about using pid/tid. for the time being, lets keep this, with the understanding that as we add support for other sandboxes, we might have to change this interface.
```suggestion
    pid: int
```